src/components/App/App.tsx

```diff
@@ -88,6 +92,7 @@ const App = () => {
               window.location.href = 'https://www.gofundme.com/f/gift-a-meal';
               return null;
           }}/>
+          <Route path="/gift-a-meal-page">{returnComponent('giftameal')}</Route>
```

Comment:
thoughts on naming the route to be just "gift-a-meal"?

Reply:
Sorry! Somehow missed all of these comments this weekend. Yes, agree this should just be gift-a-meal. That route is currently being used to link to the external GoFundMe page so I didn't want to overwrite it yet.

Reply:
Oh good catch! I didn't know about the GoFundMe route. I'll ask in the channel about the best way to disambiguate the two.

Reply:
Ok based on the Slack convo, let's just use "/gift-a-meal-page" for now until the full page is ready :)